x-pack/plugins/monitoring/server/config.test.ts

```diff
@@ -19,7 +19,9 @@ const MOCKED_PATHS = [
 
 beforeEach(() => {
   const spy = jest.spyOn(fs, 'readFileSync').mockImplementation();
-  MOCKED_PATHS.forEach((file) => when(spy).calledWith(file).mockReturnValue(`contents-of-${file}`));
+  MOCKED_PATHS.forEach((file) =>
+    when(spy).calledWith(file, 'utf8').mockReturnValue(`contents-of-${file}`)
```

Comment:
`react-when` now only matches when the `calledWith()` arguments match exactly, so needed to define the `utf8` argument.